test/integration/acceptance/add_peer_test.cpp

```diff
@@ -195,7 +195,7 @@ TEST_F(FakePeerFixture, RealPeerIsAdded) {
       proto::BlockBuilder()
           .transactions(std::vector<shared_model::proto::Transaction>{
               complete(baseTx(kAdminId).addPeer(itf_->getAddress(),
-                                                itf_->getThisPeer()->pubkey()),
+                                                itf_->getMyKey()->publicKey()),
```

Comment:
I think we'd rather change
https://github.com/hyperledger/iroha/blob/2080261f1076be64366cd73b2abb7b41cdb9d4ea/shared_model/builders/protobuf/builder_templates/transaction_template.hpp#L160-L163
to use `PublicKeyHexStringView` and remove new `getMyKey` function